Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.13.15-alpine AS builder
+
+RUN apk add --update --no-cache ca-certificates bash make gcc musl-dev git openssh wget curl
```

Comment:
I don't think the builder container would need anything other than the compiler :)